Azure/communication-ui-library-android
# [Feature] Features flags

## Purpose

* FeatureFlag and API for optional features

Supports built-in features, and optional features.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
- Enabled "diagnostics" flag for more options
- Verify that Bluetooth/ScreenShareZoom options are there
- Restart the app to ensure flags are persisted

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
